server/controllers/finance/reports/clientsReport/index.js

```diff
@@ -26,35 +25,25 @@ function document(req, res, next) {
   let report;
 
 
-  _.defaults(
-    session,
-    {
-      dateFrom : new Date(params.dateFrom),
-      dateTo : new Date(params.dateTo),
-      detailPrevious : params.detailPrevious,
-      ignoredClients : params.ignoredClients,
-      enterprise : req.session.enterprise,
-    });
-
+  _.defaults(session, { dateFrom : new Date(params.dateFrom), dateTo : new Date(params.dateTo), detailPrevious : params.detailPrevious, ignoredClients : params.ignoredClients, enterprise : req.session.enterprise });
```

Comment:
Just so you know, you can use ECMAScript2016 destructuring to assign variables inline from an object.  If you used it above, it would look like this:
```js
// destructure the parameters we care about
const { dateFrom, dateTo, detailPrevious, ignoredClients } = params;

// add them to the defaults
_.defaults(session, { dateFrom: new Date(dateFrom), dateTo: new Date(dateTo), detailPrevious, ignoredClients, enterprise = req.session.enterprise });
```
Just an FYI - I don't necessarily think it is appropriate here. :)